kroxylicious-operator/src/main/resources/META-INF/fabric8/kafka-proxies.kroxylicious.io-v1.yml

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
so the `v1` in the filename refers to the API version this file complies with rather than the version it creates?

Reply:
This stems from the `LocallyRunOperatorExtension` being opinionated:

```java
  public static void applyCrd(String resourceTypeName, KubernetesClient client) {
    String path = "/META-INF/fabric8/" + resourceTypeName + "-v1.yml";
    // ...
  }
```

The v1 thing looks like a bug to me, so I opened https://github.com/operator-framework/java-operator-sdk/issues/2561.

Reply:
Per that linked issue, the v1 in this filename is the version of the `CustomResourceDefinition` API, not the API we're declaring. So, as confusing as the file name is, we're stuck with it for now if we want to make use of the `LocallyRunOperatorExtension`, which I think we do.

Reply:
Probably worth throwing a reference to that issue in as a comment in the file as I'm definitely going to find the naming confusing at some random point in the future.

Hmm its also got annoying implications for the putative filter CR we were discussing over coffee, certainly not something we need to consider in the scope of this PR.